### DISCUSSION TITLE
Installer

### DISCUSSION THREAD

**Discussion by kfd821** [flybywiresim/aircraft#3314] [Help] [answered]
On the new installer it offers 3 ways.  Stable, Development and Experimental FBW.  My question is do Ihave to install the Experimental FBW as well as the Development or is either of them installed with the other?

**Comment by ChruutvoLuzi** [comment, marked as answer]
Experimental FBW is just Development version with a few hours delay and the custom FlyByWire

**Reply by kfd821** [reply]
Thank you, then If I want the Development version and FBW I need to install the Experimental FBW?

**Reply by robinottawa** [reply]
I don't even see Experimental. I've used Dev version (just press and install) and I now use the Custom (just press and install).

**Reply by kfd821** [reply]
Have you got the latest installer - it definitely shows Stable Development and Experimental

**Reply by ChruutvoLuzi** [reply]
> Thank you, then If I want the Development version and FBW I need to install the Experimental FBW?

yes exactly

**Reply by tracernz** [reply]
Latest installer: https://api.flybywiresim.com/installer

**Reply by kfd821** [reply]
Thank you for all your help.

**Reply by robinottawa** [reply]
My bad. It says right above "Custom". :)
